src/collection/views/Collection.tsx

```diff
@@ -207,29 +229,49 @@ const Collection: FunctionComponent<CollectionProps> = ({ match, history }) => {
 							<ToolbarRight>
 								<ToolbarItem>
 									<div className="c-button-toolbar">
-										{/* TODO add aria label once merged in components repo */}
-										<Button type="secondary" icon="bookmark" />
-										<Button type="secondary" icon="share-2" />
-										<Button type="secondary" icon="file-plus" />
-										<Button type="secondary" label="Alle items afspelen" />
-										<PermissionGuard
-											permissions={[
-												// { permissionName: 'canEditOwnCollections', obj: collection },
-												{ permissionName: 'canEditAllCollections' },
-											]}
+										<Button
+											title="Bladwijzer"
+											type="secondary"
+											icon="bookmark"
+											ariaLabel="Bladwijzer"
+										/>
+										<Button title="Deel" type="secondary" icon="share-2" ariaLabel="Deel" />
+										<ControlledDropdown
+											isOpen={isOptionsMenuOpen}
+											onOpen={() => setIsOptionsMenuOpen(true)}
+											onClose={() => setIsOptionsMenuOpen(false)}
+											placement="bottom-end"
+											autoSize
 										>
-											<PermissionGuardPass>
-												<Button
-													type="secondary"
-													icon="edit"
-													onClick={() =>
-														history.push(
-															`/${RouteParts.Collection}/${collection.id}/${RouteParts.Edit}`
-														)
-													}
+											<DropdownButton>
+												<Button type="secondary" icon="more-horizontal" />
```

Comment:
add tooltip en aria

"overige acties"

Reply:
Heb het Meer opties genoemd omdat ze het zo ook had genoemd in dat berichtje over die tooltips.